manifests/role/webhost/htvm/global_primary.pp

```diff
@@ -10,4 +10,11 @@
   include nebula::role::webhost::htvm::site_primary
   include nebula::profile::hathitrust::cron::statistics
   include nebula::profile::hathitrust::cron::catalog
+
+  cron {
+    'wordpress cron':
+      user    =>  'nobody',
```

Comment:
Looks like this is changing from libadm to nobody, which I assume is fine.

Reply:
Yeah, I figure we can generally rely on `nobody` being present, and that's the user the web server runs as anyway.